twilight-interactions-derive/src/command/model/parse.rs

```diff
@@ -284,11 +284,11 @@ impl ChannelType {
             "guild_voice" => Ok(Self::GuildVoice),
             "group" => Ok(Self::Group),
             "guild_category" => Ok(Self::GuildCategory),
-            "guild_news" => Ok(Self::GuildNews),
+            "guild_announcement" => Ok(Self::GuildAnnouncement),
```

Comment:
We should keep the old names until the next release to avoid breaking changes.
```suggestion
            "guild_announcement" | "guild_news" => Ok(Self::GuildAnnouncement),
```